.docker/app.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+chown :www-data bootstrap/cache \
+                storage/app/public \
+                storage/framework/{cache,sessions,testing,views} \
+                storage/framework/cache/data \
+                storage/logs
+
+# Copy the default environment for first run and make it writable by anyone so that it can be edited
+[ ! -f .env ] && cp .env.default .env && chmod ugo+rw .env && php artisan key:generate
+# Or just set the key if not set yet
+grep APP_KEY=CHANGEME .env > /dev/null && php artisan key:generate
```

Comment:
It should recreate it each time the container is started, not at build time. I added the `grep` line so that if you follow the instructions and copy the file manually, you still get it set up (personally, I didn't know you need to set this key as it was preconfigured on the projects I've worked on before - we could probably get away with doing that here too but whatever, this was an interesting challenge). The first line I only added because at the time I didn't take the Reddit setup into account, but yeah.

Reply:
Ah of course, my bad. I think I still had an older squire-app image around when I tested this which obviously didn't include the updated app.sh. No idea why it wasn't automatically rebuilt though.